lib/userAnalytics.js

```diff
@@ -30,7 +30,7 @@ function getCountry(params) {
 		if (!countryLookup) {
 			/*This is sync operation and can stall application if db file is large, 
 			so open file only if user have asked for county analytics*/
-			countryLookup = maxmind.openSync(path.join(__dirname, '../maxmindDb/GeoLite2-Country.mmdb'));
+      countryLookup = new maxmind.Reader(fs.readFileSync('../maxmindDb/GeoLite2-Country.mmdb'));
```

Comment:
The `fs` module is missing from the top so we would need to require it. Plus to resolve the database file itself we would need to join the `__dirname` with the location of the file like so -
`new maxmind.Reader(fs.readFileSync(path.join(__dirname, '../maxmindDb/GeoLite2-Country.mmdb')));`

Otherwise, it might result in this error - `Error: ENOENT: no such file or directory`